src/ert/config/ert_config.py

```diff
@@ -549,8 +549,9 @@ def manifest_to_json(self, iens: int = 0, iter: int = 0) -> Dict[str, Any]:
                 manifest[f"{name}_UNSMRY"] = f"{input_file}.UNSMRY"
                 manifest[f"{name}_SMSPEC"] = f"{input_file}.SMSPEC"
             if isinstance(respons_config, GenDataConfig):
-                if respons_config.report_steps and iens in respons_config.report_steps:
-                    manifest[name] = input_file.replace("%d", str(iens))
+                if respons_config.report_steps:
+                    for step in respons_config.report_steps:
+                        manifest[f"{name}_{step}"] = input_file.replace("%d", str(step))
```

Comment:
Does the key-name of this manifest dict matter? Seems a bit arbitrary and not 1-to-1 corresponding with the expected filename?

Reply:
It does not matter too much. This was a suggestion/requirement for the manifest structure to look like this, I guess the idea was to be able to match response name to output file. As you have noticed this does not aligh properly every time.

Reply:
OK so a separate issue from this then. Otherwise this looks good